api/backend/blueflood/blueflood.go

```diff
@@ -286,21 +303,23 @@ var samplerMap map[api.SampleMethod]sampler = map[api.SampleMethod]sampler{
 
 // Blueflood keys the resolution param to a java enum, so we have to convert
 // between them.
-func (c Config) bluefloodResolution(resolution int64, timestamp int64) string {
+func (c Config) bluefloodResolution(
+	desiredResolution time.Duration,
+	startMs int64) Resolution {
 	now := time.Now().UTC().Unix() * 1000
```

Comment:
Calling `.UTC()` has no effect here- it only changes the location of the time, which doesn't change the number of seconds since the Unix epoch.